dev/autotools/autogen.sh

```diff
@@ -1,18 +1,3 @@
 #!/bin/sh -e
 
-# copy TAP driver into build-aux
-automake_ver=$(automake --version | \grep -E -o '[0-9]\.[0-9]{2}')
-automake_dir=$(dirname $(which automake))/..
-
-mkdir -p build-aux
-cp -f $automake_dir/share/automake-$automake_ver/tap-driver.sh build-aux
```

Comment:
Isn't this still required? I remember autotools not putting the tap driver in the aux directory, which made the whole thing fail.

Reply:
I don't recall having any issues, the AUX_DIR macro is supposed to instruct Autotools to put its files in that directory IIRC. 

I'll try the setup from scratch once I come back to my computer and let you know.

Reply:
It works on my machine (using Arch linux).

Reply:
Thanks for confirming. I'll merge your changes as-is, thank you for the fix!